src/sentry/static/sentry/app/components/sidebar/sidebarDropdown/index.jsx

```diff
@@ -73,94 +74,113 @@ const SidebarDropdown = withApi(
+                                    >
+                                      {t('Members')}
+                                    </SidebarMenuItem>
+                                  )}
+
+                                  {hasTeamRead && (
+                                    <SidebarMenuItem to={`/settings/${org.slug}/teams/`}>
+                                      {t('Teams')}
+                                    </SidebarMenuItem>
+                                  )}
+
+                                  <Hook
+                                    name="sidebar:organization-dropdown-menu"
+                                    organization={org}
+                                  />
+
+                                  {!config.singleOrganization && (
+                                    <SidebarMenuItem>
+                                      <SwitchOrganization
+                                        canCreateOrganization={canCreateOrg}
+                                      />
+                                    </SidebarMenuItem>
+                                  )}
+
+                                  <Divider />
+                                </React.Fragment>
+                              )}
+
+                              {!!user && (
+                                <React.Fragment>
+                                  <UserSummary to="/settings/account/details/">
+                                    <UserBadgeNoOverflow user={user} avatarSize={32} />
+                                  </UserSummary>
+
+                                  <div>
+                                    <SidebarMenuItem to="/settings/account/">
+                                      {t('User settings')}
+                                    </SidebarMenuItem>
+                                    <SidebarMenuItem to="/settings/account/api/">
+                                      {t('API keys')}
+                                    </SidebarMenuItem>
+                                    {user.isSuperuser && (
+                                      <SidebarMenuItem to="/manage/">
+                                        {t('Admin')}
+                                      </SidebarMenuItem>
+                                    )}
+                                    <SidebarMenuItem
+                                      data-test-id="sidebarSignout"
+                                      onClick={this.handleLogout}
+                                    >
+                                      {t('Sign out')}
+                                    </SidebarMenuItem>
+                                  </div>
+                                </React.Fragment>
+                              )}
+                            </OrgAndUserMenu>
+                          );
+                        }}
+                      </Portal>
+                    </SidebarDropdownRoot>
+                  )}
+                </Reference>
+              </Manager>
```

Comment:
Having to use the Manager & Reference is a bit awkward. Would the interface be better if the actor was a renderProp to DropdownMenu, and `children` was put into the Portal by `DropdownMenu`?

Reply:
Yeah, I was trying to avoid changing `DropdownMenu`'s API, but I don't see a way around it.

Reply:
I agree that this feels a bit heavy on the boilerplate.

Would it be possible to have a parent component that takes care of this boilerplate? Something like `DropdownMenuGlobal` , and then we could slowly replace all instances of A with B?